com.snipaste.Snipaste.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.snipaste.Snipaste
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: snipaste
+separate-locales: false
+
+finish-args:
+  - --device=all
+
+  - --socket=wayland
+  - --socket=fallback-x11
+
+  - --share=ipc
+  - --share=network
+
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+  # D-Bus Access
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: Snipaste
+    buildsystem: simple
+    build-commands:
```

Comment:
this needs to be extra-data if redistribution is not allowed.

Reply:
fixed by using extra data